README.rst

```diff
@@ -224,6 +224,32 @@ has a value which is not in the expected range
 Example: ping is by default a critical metric which is expected to be always 1
 (reachable).
 
+``Available Checks``
+--------------------
+
+``Ping``
+~~~~~~~~
+
+This check returns information on device ``uptime``, ``downtime``,
+``RTT (Round trip time) maximum, minimum and average``.
+It creates a ``Ping`` metric to store this information and an ``AlertSettings``
+to monitor them.
+The Charts ``uptime``, ``packet loss`` and ``rtt`` are created. The ``fping``
+command is used to collect these metrics.
+You may choose to disable auto creation of this check by using the setting
+`OPENWISP_MONITORING_AUTO_PING <#OPENWISP_MONITORING_AUTO_PING>`_.
+
+``Configuration Modified``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It checks the configuration status of the device periodically and changes
```

Comment:
What about this rewording

"This will periodically check the configuration present in the device and will change the device health status to ``PROBLEM`` if the configuration is not in sync with the expected by OpenWISP. The time until this check changes the health status can be configured using the setting `OPENWISP_MONITORING_DEVICE_CONFIG_CHECK_MAX_TIME`"

Reply:
It sounds a bit technical.